src/services/fileStorage/model.js

```diff
@@ -25,6 +25,13 @@ const permissionSchema = new Schema(
 
 enableAuditLog(permissionSchema);
 
+const SecurityCheckStatusTypes = {
+	pending: 'pending',
+	verified: 'verified',
```

Comment:
why not to use uppercase letters here for constants and freeze them? https://www.javascripttutorial.net/es6/javascript-const/ 
```suggestion
	VERIFIED: 'verified',
```

Reply:
Good point